src/data/README.md

```diff
@@ -1,7 +1,23 @@
-# Data
+# OpenThaiGPT Dataset Pipeline Overview
 
 Folder `openthaigpt_pretraining_data` contains the preprocess function, EDA notebook, and examples for each dataset
 
 Folder `notebook` contains the EDA or other experiment notebook.
 
 Folder `scripts` contains the main program which will call function in folder `openthaigpt_pretraining_data`
+
+## Description
+
+The OpenThaiGPT Overview dataset pipeline is meticulously designed to fully preprocess raw data, extract essential information, and prepare the dataset in a trainable format. Here's an extended breakdown of its key components:
+
+## Workflow
+
+1. Obtain Raw Data in various formats such as CSV, PDF, etc., and convert it into plain text format.
+2. Transform plain text data from each raw data source into the JSONL format.
+3. Consolidate all JSONL datasets into a single file. This pipeline can be found at: `src/data/scripts/merge_jsonl/` to merge JSONL file
```

Comment:
Only some data such as oscar, require merge jsonl. When we merge jsonl we do it seperately for each data.


For example
- oscar
    - oscar22.jsonl
    - oscar23.jsonl
    - oscar19.jsonl
- pantip
    - pantip2g.jsonl
    - pantip3g.jsonl
    

we need to run merge jsonl code two time targeting pantip and oscar seperately.


The result will be:
- oscar
    - oscar22.jsonl
    - oscar23.jsonl
    - oscar19.jsonl
    - oscarall.jsonl
- pantip
    - pantip2g.jsonl
    - pantip3g.jsonl
    - pantipall.jsonl
    - 
    
This is ugly, but it is our current pipeline works

Reply:
Also please add path to blind pdpa code https://github.com/OpenThaiGPT/openthaigpt-pretraining/tree/main/src/data/scripts/blind_pdpa